napari/utils/translations.py

```diff
@@ -465,7 +465,10 @@ def _load_language() -> str:
     )
     if default_config_path.exists():
         with open(default_config_path) as fh:
-            data = safe_load(fh) or {}
+            try:
+                data = safe_load(fh) or {}
+            except Exception:
```

Comment:
Should I catch the specific issue you encountered and raise a Warning or similar @tlambert03 ?

Reply:
thanks for this @goanpeca!

yeah, I think if the file does exist, but there is an error in loading it, we should let the user know that their file is a) being ignored and b) will be overwritten when napari closes (is that correct?).

can you perhaps add a test that mocks the contents of `settings.yaml` with some un-parseable nonsense?  (just in case a new `safe_load` is added in the future that tries to read `settings.yaml` but forgets to wrap it?